recipe/run_test.bat

```diff
@@ -0,0 +1,384 @@
+
+:: The environment variables 'LIBSIMDPP_NAME_VERSION' is
+:: being passed through. If you are splitting build and test phases, 
+:: ensure that this value is also set similarly at test time.
+echo on
+
+if not exist %PREFIX%\Library\lib\pkgconfig\%LIBSIMDPP_NAME_VERSION%.pc exit 1
+if not exist %PREFIX%\Library\include\libsimdpp-2.1\simdpp\capabilities.h exit 1
```

Comment:
I think most of these are a little bit overkill. It's fine to check for the pc file and a couple of headers, but there is no value in checking that all headers exist.

Reply:
ok, makes sense, I tool the first, middle, and last one and the `.pc` file